src/Systems/Systems.jl

```diff
@@ -132,34 +181,58 @@ end
 
 
 """
-Type that represents a discrete-time affine system with non-deterministic inputs,
+Type that represents a discrete-time affine system with nondeterministic inputs,
 
-x_{k+1} = A x_{k} + u_{k},
+``x_{k+1} = A x_{k} + u_{k}``,
 
 where
 
-- A is a square matrix
-- x(0) ∈ X0
-- u_{k} ∈ U_{k}, where U_{k} is a piecewise-constant set-valued function defined over [t1, t1+δ], ... , [tN, tN+δ]
+- ``A ``is a square matrix
+- ``x(0) ∈ X0``
+- ``u_{k} ∈ U_{k}``, where ``U_{k}`` is a piecewise-constant set-valued function
+  defined over ``[t1, t1+δ], ... , [tN, tN+δ]`` for some δ
```

Comment:
again we don't support/handle the time shifts `t_1` at all (yet), so this is more properly ``[0, \delta], ..., [(N-1)\delta, N\delta]``.